source/features/lines-changed-per-commit.tsx

```diff
@@ -0,0 +1,53 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+
+const getCommitChanges = async (commits: string[]): Promise<string[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+				... on Commit {
+                    additions
+                    deletions
+					}
+				}
+			`).join('\n')}
+		}
+	`);
+
+	return repository;
+};
+
+function getCommitHash(commit: HTMLElement): string {
+	return (commit.dataset.channel as string).split(':')[3];
+}
+
+async function init(): Promise<void | false> {
+	const pageCommits = select.all('li.commit:not(.rgh-merge-commit)');
+	const commitChanges = await getCommitChanges(pageCommits.map(getCommitHash));
+	for (const commit of pageCommits) {
+		// @ts-ignore
```

Comment:
No need to add this, the build completes even if there are errors. They will likely be fixed before the PR is merged